core/src/validator.rs

```diff
@@ -343,6 +344,26 @@ impl Validator {
                 .set_entrypoint(entrypoint_info.clone());
         }
 
+        if let Some(ref snapshot_config) = config.snapshot_config {
```

Comment:
This is great, but can we move the trusted_validators stuff into a separate PR.  I'm thinking of something a little different, so I'd like more time to chat about that while allowing the crds stuff to land ASAP

Reply:
ok